README.md

```diff
@@ -60,13 +60,13 @@ should contain an beat/agent output, e.g. filesystem data
 ### Evaluate entire policy into output.json
```

Comment:
I couldn't comment on the unchanged line, but the file-system structure also needs to be updated (line 5-23)